File: latihan b/Latihan2 Py/Hardware.py
from Produk import Produk

#kelas yang digunakan untuk mengimplementasikan kelas Hardware
class Hardware(Produk):

	#konstruktor
	def __init__(self):
		self.brand = ""
		self.model = ""

	#getter dan setter untuk atribut brand
	def setbrand(self, brand):
		self.brand = brand

	def getbrand(self):
		return self.brand

	#getter dan setter untuk atribut model
	def setmodel(self, model):
		self.model = model

	def getmodel(self):
		return self.model

File: latihan b/latihan2 PHP/Hardware.php
<?php
// import file kelas parent
include "Produk.php";

class Hardware extends Produk{ 

/*kelas yang digunakan untuk mengimplementasikan Hardware/
/*atribut tim Hardware*/

		private $brand = "";		//brand
		private $model = "";		//model
	
		
		 // constructor dengan parameter
        public function __construct($brand ="", $model =""){
            $this->brand = $brand;
            $this->model = $model;
        }
	
	/*mengeset nilai brand*/
	public function setbrand($brand)
	{
		$this->brand = $brand;
	}
	/*mengeset nilai model*/
	public function setmodel($model)
	{
		$this->model = $model;
	}

	/*mengembalikan nilai brand*/
	public function getbrand()
	{
		return $this->brand;
	}
	/*mengembalikan nilai model*/
	public function getmodel()
	{
		return $this->model;
	}
	
	function __destruct(){
		/*destuktor*/
	}
	
}

?>